cypress/integration/landing_page/test_products_menu.js

```diff
@@ -0,0 +1,25 @@
+describe('Products menu', () => {
+  before(() => {
+    cy.setupMetamask();
+  });
+
+  beforeEach(() => {
+    cy.impersonateUser();
+  });
+
+  afterEach(() => {
+    cy.logout();
+  });
```

Comment:
are these needed for this test?

Reply:
I don't think the `afterEach` is necessary, but we definitely need the `impersonateUser`

Reply:
If you have to log in for this page, then you will need to make sure to log out after the test. If you need the impersonateUser then keep the logout. Cypress will keep that user in session and it could cause state issues for the next test. I expected the grants page to be public.

Reply:
Ah that makes sense. I'll put it back, but I just tested it on the production site, and the pages are public. The test breaks without `impersonateUser()` though.